<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_dlg</class>
 <widget class="QDialog" name="login_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>149</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>228</width>
    <height>149</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>228</width>
    <height>149</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Auth error</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons/emule/emule.qrc">
    <normaloff>:/emule/TrayConnected.ico</normaloff>:/emule/TrayConnected.ico</iconset>
  </property>
  <widget class="QLabel" name="lbl_username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="buddy">
    <cstring>edtUsername</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPasswd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtUsername">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_passwd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>68</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>68</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="buddy">
    <cstring>edtPasswd</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Input login and password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>edtUsername</tabstop>
  <tabstop>edtPasswd</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="Icons/emule/emule.qrc"/>
 </resources>
 <connections/>
</ui>
